modules/ROOT/pages/redelivery-policy.adoc

```diff
@@ -0,0 +1,63 @@
+= Redelivery Policy
```

Comment:
What *about* redelivery policy does this topic focus on? What it is? How you use it? 
There might not be a more specific title, but consider whether that is the case.

Reply:
I agree, but it is hard to change the name of one topic and have it not go with the others in the group. so I think we should do this by topic group.